src/main/java/com/incentives/piggyback/user/UserApplication.java

```diff
@@ -6,13 +6,12 @@
 import org.springframework.web.bind.annotation.RestController;
 
 @SpringBootApplication
-@RestController
 public class UserApplication {
 
-	@RequestMapping("/")
-	public String home() {
-		return "Hello World on Docker";
-	}
+//	@RequestMapping("/")
+//	public String home() {
+//		return "Hello World on Docker";
+//	}
```

Comment:
Probably just remove this rather than commenting out.

Reply:
okay , will do it.